acceptance/bundle/paths/fallback/output.txt

```diff
@@ -0,0 +1,76 @@
+
+>>> $CLI bundle validate -t development -o json
+
+Exit code: 0
+
+>>> $CLI bundle validate -t error
+Error: notebook this value is overridden not found. Local notebook references are expected
+to contain one of the following file extensions: [.py, .r, .scala, .sql, .ipynb]
+
+Name: fallback
+Target: error
+Workspace:
+  User: tester@databricks.com
+  Path: /Workspace/Users/tester@databricks.com/.bundle/fallback/error
+
+Found 1 error
+
+Exit code: 1
+
+>>> jq -e 
+.jobs.my_job.tasks[0].task_key == "notebook_example" and
+(.jobs.my_job.tasks[0].notebook_task.notebook_path | endswith("/src/notebook"))
+ output.development.json
+true
```

Comment:
I don't mind having this kind of assertions once in a while, but if there are 100s of tests using them it would become more difficult to maintain than auto-updatable golden files, so we should have a clear plan where we add them and where we rely on golden files.

Why did you opt for translating the assertions from go to jq here instead of relying on golden files?

As an alternative, would a more targetted golden file help to replace these (e.g. jq .jobs.my_job.tasks)? This should be complemented by a comment that explains what to look for: "all source files should be placed inside src/ subdirectory).

Reply:
I removed them in favor of more narrow golden files.